doc/classes/VisualShaderNode.xml

```diff
@@ -27,6 +29,7 @@
 			<argument index="0" name="values" type="Array">
 			</argument>
 			<description>
+				Sets the default input ports using the [Array] filled with [code][index, value][/code]. For example: [0, Vector3(0, 0, 0), 1, Vector3(0, 0, 0)].
```

Comment:
Remember to enclose the array in `[code] ... [/code]`. It's missing for the second one.